build-system/test-configs/forbidden-terms.js

```diff
@@ -1061,7 +1061,11 @@ const forbiddenTermsSrcInclusive = {
   '\\.indexOf\\(.*===?.*\\.length': 'use endsWith helper in src/string.js',
   '/url-parse-query-string': {
     message: 'Import parseQueryString from `src/url.js`',
-    allowlist: ['src/url.js', 'src/mode.js'],
+    allowlist: [
+      'build-system/tasks/check-types.js',
```

Comment:
Are you still using `indexOf` in `check-types.js`? If it's a nodejs string operation, I'd imagine you can use `.includes`, `.startsWith`, etc.?

Reply:
I'm so confused...how does this question relate to this PR?

The added line you've commented on is due to my typechecking a filename that is "forbidden" to use.

Reply:
Crap, I misread this. (I thought this had to do with `indexOf` on line 1061, but it's really about `url-parse-query-string` on line 1062. Carry on.

Reply:
I was also very confused 😆